File: old_contents/boj_codes/C++17/2588.cpp
// https://www.acmicpc.net/problem/2588
// First Written : 20210107
// Last Modified : 20210107

#include <cstdio>

int main(){
    int a, b, copied_b, digit;

    scanf("%d %d", &a, &b);
    copied_b = b;

    while (copied_b != 0){
        digit = copied_b % 10;
        printf("%d\n", a * digit);

        copied_b /= 10;
    }

    printf("%d\n", a * b);
    
    


    return 0;
}

File: old_contents/boj_codes/Python3/2588.py
#https://www.acmicpc.net/problem/2588
# 20200510

import sys

a = int(sys.stdin.readline().rstrip())
b = int(sys.stdin.readline().rstrip())

num = b

while num != 0:
    last = num % 10
    print(a * last)

    num //= 10

print(a * b)

